lib/pbench/server/database/database.py

```diff
@@ -14,16 +14,19 @@ class Database:
 
     @staticmethod
     def get_engine_uri(config, logger):
+        engine_uri = None
         try:
-            psql = config.get("Postgres", "db_uri")
-            return psql
-        except (NoSectionError, NoOptionError):
-            msg = "Failed to find a [Postgres] section in configuration file."
+            engine_uri = config.get("database", "db_uri")
+        except NoSectionError:
+            msg = "Failed to find [database] section in configuration file."
+        except NoOptionError:
+            msg = "Failed to find 'db_uri' value in [database] section of configuration file."
```

Comment:
`NoSectionError` and `NoOptionError` report the config section/option which was missing...we should consider using those values here instead of hard-coding the message (had we done that, it might have made this code change smaller...).

Or, better yet, we should have `str(NoSectionError)` produce the appropriate message, so that we don't have to do it here (and elsewhere).